pkg/preparer/orchestrate.go

```diff
@@ -131,6 +131,15 @@ func (p *Preparer) handlePods(podChan <-chan pods.PodManifest, quit <-chan struc
 			manifestLogger.NoFields().Debugln("New manifest received")
 
 			working = p.verifySignature(manifestToLaunch, manifestLogger)
+			if !working {
+				err = p.hooks.RunHookType(hooks.AFTER_AUTH_FAIL, pods.NewPod(manifestToLaunch.ID(), pods.PodPath(p.podRoot, manifestToLaunch.ID())), &manifestToLaunch)
+				if err != nil {
```

Comment:
Previously there were two methods `RunBeforeHooks` and `RunAfterHooks`. I removed them in favor of the general `RunHookType`, but it does make specializing the data we export difficult. I'm open to revisiting that pattern.